### ISSUE TITLE
Add Flask example project

### ISSUE THREAD

**Issue by dmitry-viskov** [dmitry-viskov/pylti1.3#15]

**Comment by ssilverm** [comment]
I'm still working on this.  I'm still stuck at state not found (via cookies, its still in the request object).  HOpefully the newest changes fix that.

**Comment by dmitry-viskov** [comment]
hi @ssilverm .

I may assume that bug is fixed: https://github.com/dmitry-viskov/pylti1.3/commit/b7adfae6f56a8624390dff9fc7baef401f8ea506

Here is Flask usage example: https://github.com/dmitry-viskov/pylti1.3-flask-example
I've checked - It was working fine for my test cases.

**Comment by ssilverm** [comment]
Yep, I got my version working too after the cookie issue was fixed.

**Comment by ssilverm** [comment]
Your example shows off everything the library can do, I've made a very minimal example just to get through the launch.

```
from flask import Flask, request, session, render_template
import json
from werkzeug.utils import redirect
from pylti1p3.contrib.flask import (FlaskCookieService, FlaskOIDCLogin,
                                    FlaskRequest, FlaskSessionService, FlaskMessageLaunch)
from pylti1p3.tool_config import ToolConfJsonFile
tool_conf = ToolConfJsonFile('data.json')

app = Flask(__name__)
app.config['SECRET_KEY'] ='CHANGEME'
app.config.update(
    SESSION_COOKIE_SECURE=True,
    SESSION_COOKIE_SAMESITE='None',
)

@app.route('/login/', methods=['POST', 'GET'])
def login():

    new_request = FlaskRequest(
        request_data=request.data,
        request_is_secure=request.is_secure,
        cookies=request.cookies,
        session=session
    )

    oidc_login = FlaskOIDCLogin(
        request=new_request,
        tool_config=tool_conf,
        session_service=FlaskSessionService(new_request),
        cookie_service=FlaskCookieService(new_request)
    )

    return oidc_login.redirect(new_request.get_param('target_link_uri'))

@app.route('/launch/', methods=['POST', 'GET'])
def launch():
    new_request = FlaskRequest(
        request_data=request.data,
        request_is_secure=request.is_secure,
        cookies=request.cookies,
        session=session
    )

    message_launch = FlaskMessageLaunch(
        request=new_request,
        tool_config=tool_conf,
        session_service=FlaskSessionService(new_request),
        cookie_service=FlaskCookieService(new_request)
    )

    given_name = message_launch.get_launch_data().get('given_name')
    print(json.dumps(message_launch.get_launch_data(), indent=2))
    
    return render_template(
        "launch.html",
        given_name=given_name
    )

```

**Comment by dmitry-viskov** [comment]
Flask adapter already available in `1.4.0` version: https://pypi.org/project/PyLTI1p3/1.4.0/
